config/common/group_vars/k8s-cluster/ck8s-k8s-cluster.yaml

```diff
@@ -0,0 +1,94 @@
+kube_oidc_auth: true
+kube_oidc_url: "https://dex.BASE-DOMAIN"
+kube_oidc_client_id: "kubelogin"
+kube_oidc_username_claim: "email"
+
+kube_apiserver_enable_admission_plugins:
+  - "PodSecurityPolicy"
+  - "NamespaceLifecycle"
+  - "LimitRanger"
+  - "ServiceAccount"
+  - "DefaultStorageClass"
+  - "DefaultTolerationSeconds"
+  - "MutatingAdmissionWebhook"
+  - "ValidatingAdmissionWebhook"
+  - "ResourceQuota"
+  - "NodeRestriction"
+
+kube_kubeadm_apiserver_extra_args:
+  delete-collection-workers: "3"
+
+kube_proxy_mode: iptables
+kube_proxy_metrics_bind_address: "0.0.0.0:10249"
+
+enable_nodelocaldns: false
+
+kubernetes_audit: true
+audit_log_path: "/var/log/audit/kube-apiserver-audit.log"
```

Comment:
Could you open an issue in the apps repo so that we remember to update fluentd to look for this file?

Reply:
https://github.com/elastisys/compliantkubernetes-apps/pull/211